data-structures-linked-lists/remove-next.js

```diff
@@ -0,0 +1,9 @@
+/* exported removeNext */
+
+function removeNext(list) {
+  if (list.next === null) {
+    return list;
```

Comment:
`removeNext` should not return anything:

---

![image](https://user-images.githubusercontent.com/16143771/156837656-55a3de36-365b-4841-aa13-7391def70d73.png)